rosidl_generator_py/resource/_msg.py.em

```diff
@@ -87,6 +87,13 @@ for member in message.structure.members:
 @[end if]@
 @#>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 
+# This is being done at the module level and not on the instance level to avoid looking
+# for the same variable multiple times on each instance. This variable is not supposed to
+# change during runtime so it makes sense to only look for it once.
+from os import getenv
+
+ros_python_check_fields = getenv("ROS_PYTHON_CHECK_FIELDS", default='')
```

Comment:
```suggestion
ros_python_check_fields = getenv('ROS_PYTHON_CHECK_FIELDS', default='')
```

Reply:
Also, when I run the tests on this, flake8 says that the imports are in the wrong order.  So this probably needs to move above somehow.

Reply:
That's weird, when I run the test it isn't showing me that. It did before and I moved it up and stop appearing the error. I'll check.

Reply:
You have to run one of the tests for the generated output.  I did:

```
colcon test --event-handlers console_direct+ --packages-select visualization_msgs
```

Reply:
I see the problem now. I was running the test for only this package instead of the whole build. The problem is that for actions and services the `_msg.py.em` template is being called multiple times and the import appears multiple times as well. I think a solution would be to move that import up one module level and import it on the `_idl.py.em` module. Let me know what you think.